src/app/index.config.js

```diff
@@ -16,4 +16,11 @@ export function config ($logProvider, toastrConfig, $mdThemingProvider, cfpLoadi
 
   // Load Bar
   cfpLoadingBarProvider.includeSpinner = false;
+
+  // Google+
+  GooglePlusProvider.init({
+    clientId: '867651060370-4oqj1vdb06deotdvek82riivej3q4pmk.apps.googleusercontent.com'
```

Comment:
Can you set the `scopes` to `profile` here?  Details can be found at https://github.com/mrzmyr/angular-google-plus/issues/6 with a specific example at https://github.com/mrzmyr/angular-google-plus/issues/6#issuecomment-52639212.

Right now it is requesting the `https://www.googleapis.com/auth/plus.login` scope which is over-asking.